source/stylesheets/main.sass

```diff
@@ -61,5 +61,3 @@
 @import views/subscriptions/plans
 @import views/teachers/index
 @import views/teachers/show
-
-@import url('https://fonts.googleapis.com/css?family=Nunito:300,400,400i,700')
```

Comment:
The same here... we will import this font on demo page to show the final state of componentes?

Reply:
Yes, the idea is to import the font when we do the styleguide documentation.